lib/config.js

```diff
@@ -250,10 +258,12 @@ Config.prototype.buildArgs = function () {
 Config.prototype.shouldSign = function () {
   // it doesn't make sense to sign debug builds because the restrictions on loading
   // dynamic libs prevents them from working anyway
-  return this.mac_signing_identifier !== '' &&
-         !this.skip_signing &&
+  return !this.skip_signing &&
          this.buildConfig === 'Release' &&
-         this.targetOS !== 'ios'
+         (this.mac_signing_identifier !== '' &&
```

Comment:
can you add nested `()` here for ios vs android with the `||`?

Reply:
I know it works fine because of operator precedence, but it helps readability

Reply:
Done